public/app/plugins/datasource/influxdb/types.ts

```diff
@@ -61,4 +61,14 @@ export interface InfluxQuery extends DataQuery {
   rawQuery?: boolean;
   query?: string;
   alias?: string;
+  // for migrated InfluxQL annotations
+  // query?: string; // exists in the tag but duplicated above
```

Comment:
I'm a little confused by this comment, is this something we want to include in the PR?

Reply:
I can remove this. I added this because I was migrating annotation properties to the query, and the query property was in both the InfluxQuery and DataQuery.